### PR TITLE
Add acts 2-5, with a little dialogue

### PR DESCRIPTION
**Pull request by KPetrucci**
I added a bit more to the play. Acts 2-5 describe the goings-on from the coach telling the team members to write in the notebook, and the robot's program having some issues.

### CODE REVIEW

File: Scene_1/Act_5.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+Enter MRS. PETRUCCI from upstairs
+
+MRS. PETRUCCI: Are your notebook entries finished?
+VICTORIA: Yup
+KATIE: Um.. not quite
+BEN: Yeah, we got a bit distracted programming
+KATIE: Is it ok if we finish at home?
+MRS. PETRUCCI: Fine. But if you don't do it this week, you have to do it first thing next meeting. Got it?
+ALL: Yes
+MRS. PETRUCCI: What's the robot doing? (ROBOT still spinning in circles, now repeatedly saying "Rhyme Know Reason it LIT")
```
Comment (ItsHarper) [inline Scene_1/Act_5.txt]: should be "is lit"

Comment (ItsHarper) [reply Scene_1/Act_5.txt]: You start these comment threads by hovering over the line of code you want to comment on and pressing the plus button that pops up

**Review by ItsHarper — CHANGES_REQUESTED** [review]
When someone submits a pull request, you can review their changes. You make comments on particular lines of code, and can approve the PR or demand that changes be made to it before getting merged.

**Comment by ItsHarper** [comment]
I was pointing out a typo, not trying to change the capitalization

**Comment by KPetrucci** [comment]
Oh. Thanks

________________________________

**Comment by FTCBen** [comment]
Typos fixed.

**Pull request merged**